### ISSUE TITLE
Paragraph formatting in editor

### ISSUE THREAD

**Issue by Nedayvozov** [nickmilo/Cybertron#8]
It's not possible to add paragraphs to plain text.

I've tried to add the following class, but it's doesn't work:
`.markdown-preview-view br { content: ''; display: block; margin-top: 10px; }`

Could you please add this feature to your project or suggest me how I can add it?

**Comment by nickmilo** [comment]
Are you copy and pasting? One thing I do is hit cmd-opt-shift-v to paste
(in Mac) to ensure it doesn't paste in a weird way.

**Comment by Nedayvozov** [comment]
Yes. I've just followed the instuction from [this topic](https://forum.obsidian.md/t/paragraph-formatting-in-editor/16390) and copied and pasted the following code to the end of the css file.
.markdown-preview-view br { content: ''; display: block; margin-top: 10px; }
but this doesn't work for your project, so I decided to ask you directly.
